README.md

```diff
@@ -97,13 +99,59 @@ by setting
 c.LatexConfig.bib_command = '<custom_bib_command>'
 ```
 
+New: `BibTeX` compilation is skipped if the following conditions are present:
+
+- `c.LatexConfig.disable_bibtex` is explicitly set to `True` in the `jupyter_notebook_config.py` file
+- There are no .bib files found in the folder
+
 To render references (`\ref{...}`), such as equation or chapter numbers, you would
 need to compile in multiple passes by setting
 
 ```python
 c.LatexConfig.run_times = 2
 ```
 
+New: `Manual Command Arguments`
+For more advanced customizations, a complete command sequence can be specified using the `manual_cmd_args` configuration in the `jupyter_notebook_config.py` file. This allows to define the exact command and use options the extension will finally execute:
+
+```python
+c.LatexConfig.manual_cmd_args = [
+    'lualatex',  # Specify the LaTeX engine (e.g., lualatex, pdflatex)
+    '-interaction=nonstopmode',  # Continue compilation without stopping for errors
+    '-halt-on-error',  # Stop compilation at the first error
+    '-file-line-error',  # Print file and line number for errors
+    '-shell-escape',  # Enable shell escape
+    '-synctex=1',  # Enable SyncTeX for editor synchronization
+    '{filename}.tex'  # Placeholder for the input file name
+]
+```
+
+The `{filename}` placeholder will be replaced by the name of the LaTeX file during compilation.
```

Comment:
```suggestion
The only supported placeholder in the manual compile command is `{filename}`. It will be replaced by the name of the LaTeX file during compilation.
```